_studio/mfx_lib/shared/include/mfx_h264_encode_struct_vaapi.h

```diff
@@ -306,7 +306,11 @@ typedef struct tagENCODE_CAPS
             UINT    DDRStreamingBufferSupport     : 1;
             UINT    LowDelayBRCSupport            : 1; // eFrameSizeTolerance_ExtremelyLow (Low delay) supported
             UINT    AVBRBRCSupport                : 1;
-            UINT                            : 11;
+            UINT    MaxNumDeltaQPMinus1           : 4;
```

Comment:
As I know, we cant modify this struct, because it is from ddi.
I think you can try:
https://github.com/Intel-Media-SDK/MediaSDK/blob/e39cd0f6d7561e2e6be1640144979866ed84db7e/_studio/mfx_lib/shared/include/mfx_h264_encode_struct_vaapi.h#L318

Reply:
@dmitryermilov : I am not sure. Can you, comment?

Reply:
I assume the author simply sync'ed with Windows, not invented a new field by himself. No concerns from me here.